File: src/main/java/ordilov/randomplay/playlist/domain/youtube/YoutubePlaylists.java
package ordilov.randomplay.playlist.domain.youtube;

import lombok.Getter;

@Getter
public class YoutubePlaylists {

  String kind;
  String etag;
  String id;
  String snippet;
  Status status;
  ContentDetails contentDetails;
  Player player;
  String[] tags;

  @Getter
  private static class Status {
    String privacyStatus;
  }

  @Getter
  private static class ContentDetails {

    int itemCount;
  }

  @Getter
  private static class Player {

    String embedHtml;
  }
}

File: src/main/java/ordilov/lissn/playlist/domain/youtube/YoutubePlaylists.kt
package ordilov.lissn.playlist.domain.youtube

data class YoutubePlaylists(
    val kind: String,
    val etag: String,
    val id: String,
    val snippet: String,
    val status: Status,
    val contentDetails: ContentDetails,
    val player: Player,
    val tags: Array<String>,
) {
    data class Status(
        val privacyStatus: String
    )

    data class ContentDetails(
        val itemCount: Int
    )

    data class Player(
        val embedHtml: String
    )

}